topdown/exported_test.go

```diff
@@ -49,6 +49,17 @@ type opt func(*Query) *Query
 
 func testRun(t *testing.T, tc cases.TestCase, opts ...opt) {
 
+	for k, v := range tc.Env {
```

Comment:
Is there something blocking the use of `t.Setenv` here? 🤔

Reply:
Ha! No. I just wrote this before discovering that function, and missed to update. Will fix.